### DISCUSSION TITLE
无线网卡有点搞清楚了

### DISCUSSION THREAD

**Discussion by fanzhong-cn** [istoreos/istoreos#679] [Q&A]
用无线网卡增加AP功能实现了，我用的是pve 虚拟机，用pci转接卡可以，如果直接装在minipcie 就不行，出现直通的地址和istoreos 自认地址不一样，技术不行搞不来。

**Comment by jjm2473** [comment]
关键看网卡型号吧

**Reply by fanzhong-cn** [reply]
型号不是已经解决了吗，按开发大佬的指定网卡就行。
